services/version.js

```diff
@@ -86,8 +121,18 @@ function latestMaybeSemVer(versions, pre) {
   return version
 }
 
-// Given a list of versions (as strings), return the latest version.
-// Return undefined if no version could be found.
+/**
+ * Finds the largest version number lexicographically or semantically from an array of strings representing version numbers and returns it as a string.
```

Comment:
Can we differentiate `latest()` and `latestMaybeSemVer()` a bit more. It isn't really clear from the docstrings what the difference is.

`latestMaybeSemVer()` is used for versions that match some kind of dotted version pattern. `latest()` is looser and will attempt to make sense of anything, falling back to alphabetic sorting.

Also worth noting that `latest()` is the one we should normally use when adding version badges.